qemu/tests/cfg/create_large_img.cfg

```diff
@@ -16,4 +15,16 @@
         - over_ext4:
             file_sys = "ext4"
             image_size_large = 16T
-            err_info = "The image size is too large for file format 'raw'"
+            qcow2:
+                image_size_large = 2049T
+            err_info = "The image size is too large for file format;"
+            err_info = '${err_info}Could not resize file: File too large;'
+            err_info = '${err_info}The requested file size is too large'
```

Comment:
or we can
error_info = xxx
error_info += yyy

either is OK for me, just fyi

Reply:
Got it, thanks.